ext/cl_khr_command_buffer_mutable_dispatch.asciidoc

```diff
@@ -0,0 +1,980 @@
+  not set on _command_ recording and _num_args_ or _num_svm_args_ is non-zero.
+
+* `CL_INVALID_OPERATION` if the `CL_MUTABLE_DISPATCH_EXEC_INFO_KHR` property was
+  not set on _command_ recording and _num_exec_infos_ is non-zero.
+
+* `CL_INVALID_VALUE` if _arg_list_ is `NULL` and _num_args_ > 0, or _arg_list_
+  is not `NULL` and _num_args_ is 0.
+
+* `CL_INVALID_VALUE` if _arg_svm_list_ is `NULL` and _num_svm_args_ > 0, or
+  _arg_svm_list_ is not `NULL` and _num_svm_args_ is 0.
+
+* `CL_INVALID_VALUE` if _exec_info_list_ is `NULL` and _num_exec_infos_ > 0, or
+  _exec_info_list_ is not `NULL` and _num_exec_infos_ is 0.
+
+The function
+indexterm:[clGetMutableCommandInfoKHR]
+[source, c]
+----
+cl_int clGetMutableCommandInfoKHR(
+    cl_mutable_command_khr command,
+    cl_mutable_command_info_khr param_name,
+    size_t param_value_size,
+    void* param_value,
+    size_t* param_value_size_ret);
+----
+Queries information about the _command_ object.
+
+_command_ Specifies the mutable-command object being queried.
+
+_param_name_ Specifies the information to query. The list of supported
+_param_name_ types and the information returned in _param_value_ by
+*clGetMutableCommandInfoKHR* is described in the
+<<mutable-command-object-queries, Mutable Command Object Queries>> table.
+
+_param_value_size_ Is used to specify the size in bytes of memory pointed to by
+_param_value_. This size must be ≥ size of return type as described in the
+<<mutable-command-object-queries, Mutable Command Object Queries>> table.
+
+_param_value_ Is a pointer to memory where the appropriate result being queried
+is returned. If _param_value_ is `NULL`, it is ignored.
+
+_param_value_size_ret_ Returns the actual size in bytes of data being queried
+by _param_name_. If _param_value_size_ret_ is `NULL`, it is ignored.
+
+[[mutable-command-object-queries]]
+._Mutable Command Object Queries_
+[width="100%",cols="<33%,<17%,<50%",options="header"]
+|====
+| Mutable Command Info
+| Return Type
+| Description
+
+| `CL_MUTABLE_COMMAND_COMMAND_QUEUE_KHR`
+| `cl_command_queue`
+| Return the command-queue associated with _command_. If `NULL` was passed as
+  the queue when _command_ was recorded, then the queue associated with the
+  command-buffer that _command_ belongs to is returned.
+
+| `CL_MUTABLE_COMMAND_COMMAND_BUFFER_KHR`
+| `cl_command_buffer_khr`
+| Return the command-buffer associated with _command_.
+
+| `CL_MUTABLE_COMMAND_COMMAND_TYPE_KHR`
+| `cl_command_type`
+| Return the command-type associated with _command_.
+
+  The list of supported event command types defined by *clGetEventInfo* is used
+  with the matching command.
+
+| `CL_MUTABLE_DISPATCH_PROPERTIES_ARRAY_KHR`
+| `cl_ndrange_kernel_command_properties_khr[]`
+| Return the properties argument specified on _command_ recording with
+  *clCommandNDRangeKernel*.
+
+  If the properties argument specified on creation of _dispatch_ was not
```

Comment:
```suggestion
  If the properties argument specified on creation of _command_ was not
``

I'm not certain on this change, but I can't see what `dispatch` is referring to otherwise.